app/lib/config/config.go

```diff
@@ -33,6 +33,8 @@ type CothoritydConfig struct {
 	Public  string
 	Private string
 	Address network.Address
+	TLSCert string
+	TLSKey  string
```

Comment:
> these are the PEM-encoded data, so string makes more sense here.

OK. I was (and still am a little) surprised because everywhere `TLSCert` is used you need to cast it to `[]byte`... Why is that? But of course, if it is just a base64 encoded string, it needs to be saved as a string 👍